src/components/Button/Button.scss

```diff
@@ -381,202 +388,48 @@ $stacking-order: (
   &.sizeLarge {
     margin: (-1 * $large-vertical-padding) (-1 * spacing(loose));
   }
+
   &.iconOnly {
-    @include recolor-icon(color('ink', 'lighter'));
+    @include recolor-icon(var(--p-icon));
```

Comment:
This diff is causing the percy conflicts. It looks like a change that was missed during the original rollout.